python_modules/libraries/dagster-dbt/dagster_dbt/asset_decorator.py

```diff
@@ -20,7 +22,9 @@ def dbt_assets(
     """Create a definition for how to compute a set of dbt resources, described by a manifest.json.
 
     Args:
-        manifest (DbtManifest): The wrapper of a dbt manifest.json.
+        manifest (Union[Mapping[str, Any], DbtManifest, Path]): A path to or contents of a dbt
+            manifest.json file that contains the dbt models, seeds, snapshots, etc. that we want to
+            create Dagster asset definitions for.
```

Comment:
This is an odd sentence.

```suggestion
        manifest (Union[Mapping[str, Any], DbtManifest, Path]): The contents of a dbt
            manifest.json file that contains the representation of a dbt project (models, tests, macros, etc). We use this representation to create corresponding Dagster assets. Alternatively, the path to a manifest.json can be specified.
```